lib/ImageEditor.js

```diff
@@ -44,6 +44,11 @@ type ImageCropData = {
     cover: string,
     stretch: string,
   }>,
+
+  /**
+   * (Optional) if true, will disable the use of external cache.
+   * */
+  useInternalCache?: boolean,
```

Comment:
This description also indicates it will default to false (which is fine once we do 😄 )

Reply:
Fixed :)